qiskit_experiments/characterization/t2ramsey.py

```diff
@@ -99,3 +102,203 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
             circuits.append(circ)
 
         return circuits
+
+
+class RamseyXY(BaseExperiment):
+    r"""Ramsey experiment for frequency calibration.
```

Comment:
Good point. Although this is mainly used for calibration, we can say this is a characterization of frequency error with sign. This is definitely a subspecies of Ramsey experiment, and having them in the same file make sense to me. I think the main point here is purpose vs experiment technique.

Another solution would be importing this in the calibration module init so that end-user can get this experiment both from characterization and calibration package. From coding point of view, keeping this experiment in `t2ramsey.py` is more reasonable, because if we need to add some feature to family of Ramsey experiment, this structure will make maintenance easier.

Reply:
I suggest changing the docstring and you are fine.
```suggestion
    r"""Ramsey experiment to measure the transition frequency.
```